modules/comments/base.php

```diff
@@ -277,18 +277,4 @@ public function set_comment_cookies( $comment_id ) {
 			setcookie( 'comment_author_url_'   . COOKIEHASH, esc_url($comment->comment_author_url), time() + $comment_cookie_lifetime, COOKIEPATH, COOKIE_DOMAIN );
 		}
 	}
-
-	/**
```

Comment:
Banned domain seems like a more direct and safer approach, sure. Sorta begs the question why we call Photon functions with a banned domain in the first place (at least assuming we know it would be one, as is the case here).

The explicit whitelisting is interesting though... because the filter it's attached to is related to query arguments being preserved, not unlike the issue we're seeking to resolve here. But the case we're encountering is not that the URL passed through `jetpack_photon_url` needs query arguments but rather the URL it redirects to (only determined Photon server-side). It may be the case that this redirect had not always existed?